### DISCUSSION TITLE
effort / luck ? any way to see it??

### DISCUSSION THREAD

**Discussion by hjonathan2019** [coinfoundry/miningcore#1479] [General]
as the tittle says, any way to see effort / luck ???

**Comment by MiningCryptoLive** [comment]
learn to read the discussion forum. There are multiple posts about this

**Reply by hjonathan2019** [reply]
in no post did I find the solution to visualize the effort of the pool
![image](https://user-images.githubusercontent.com/59586038/197409359-36454571-dd89-4112-bb99-271fdab02fc1.png)

**Comment by tripflex** [comment]
This is easy, just convert the effort value to a percentage `effort * 100`
